contributing/STYLE.md

```diff
@@ -0,0 +1,171 @@
+
+## About_ file formatting
+We are changing the way we process about_ files.
+About_* files are being reformatted for the best compatibility across PowerShell versions and our publishing tools.
+For details, see [issue #1806][issue1806].
+Please do not alter the formatting of about_ files without checking in with a repo maintainer.
+
+Basic formatting guidelines:
+
+- Limit lines to 80 characters
+- Code blocks should be limited to 76 characters
+- Within a paragraph, the following characters must be escaped using a leading `\` character: $,\`,\<
+- Tables need fit withing 76 characters
+  - Manually wrap contents of cells across multiple lines
+  - Use opening and closing `|` characters on each line
+  - See a working example in [about_Comparison_Operators][about-example]
+
+## Use semantic line breaks
+
+Add a single newline after each sentence or a clause boundary near the 100 character limit.
+This simplifies the command-line output of diffs and history.
+
+* You must include semantic line breaks
+* Limit lines to 100 characters
+* If a sentence is longer than 100 characters, break the line at a clause boundary
+  near the 100 character limit
+
+For a good explanation of semantic line breaks, see [Semantic Linefeeds by Rhodes Mill][semantics].
+
+This is not currently adopted across all of PowerShell-Docs, but we will be working towards it over time.
+Feel free to help out.
+
+## Blank lines & Spaces vs Tabs
+
+Remove duplicate blank lines.
+Multiple blank lines render as a single blank line in HTML.
+Blank lines can also signal the end of a block in Markdown.
+There should be a single blank between Markdown blocks of different types (for example,
+between a paragraph and a list).
+
+**NOTE:** Spacing is significant in Markdown.
+Always uses spaces instead of hard tabs.
+Remove extra spaces at the end of lines.
+
+## Titles/Headings
+
+Only use [ATX headings][atx] (as opposed to = or \- style headers).
+
+* Titles/headings should be surrounded by blank lines
+* Only the first letter of a title and any proper nouns in that title should be capitalized
+* Only one H1 per document
+
+When editing reference content, the H2s are prescribed by platyPS and must not be added or removed.
+Adding or removing H2 causes a build break.
+
+## Formatting syntax elements
+
+* PowerShell cmdlets are "[Pascal Cased][pascal-case]".
+  Verbs are seperated from nouns by a hyphen.
+
+* When talking about a cmdlet in paragraph, use \` to highlight cmdlet names
+
+* Property names and parameter names should be in **bold**
+
+  * For wxample:
+
+    This `Write-Host` cmdlet uses the **-Object** parameter to ...
+
+* When writing an article (as opposed to reference content), the first instance of a cmdlet name should be a link to the cmdlet documentation.
+
+## Formatting code blocks
+
+* All PowerShell syntax blocks should use &#96;&#96;&#96;powershell code fence marker.
+
+* Do not start PowerShell commands with the PowerShell prompt. For example, "`PS C:\>`".
```

Comment:
can you bold the "not"? also, "for example" is misleading - the example is to **not** do this. Maybe "Do not start the PowerShell commands withthe PowerShell prompt ("`PS C:\>`")